plugins/pencilblue/controllers/actions/forgot_password.js

```diff
@@ -41,9 +41,10 @@ module.exports = function ForgotPasswordControllerModule(pb) {
         this.userService = new UserService(this.getServiceContext());
 
         var ctx = this.getServiceContext();
+        ctx.onlyThisSite = false; // Use the global email settings in a multisite environment
         ctx.userService = this.userService;
-        ctx.siteService = new SiteService(this.getServiceContext());
-        ctx.emailService = new EmailService(this.getServiceContext());
+        ctx.siteService = new SiteService(ctx);
+        ctx.emailService = new EmailService(ctx);
```

Comment:
I'd like to ask that we don't make the change to use the same context object.  The reason is because at some point down the road the context object could be modified and effect one or more of the services' behaviors.  Providing each instance with its own service context prevents bad things from happening if one service misbehaves.